README.md

```diff
@@ -172,5 +166,5 @@ At the core, `dask-sql` does two things:
 - translate the SQL query using [Apache Calcite](https://calcite.apache.org/) into a relational algebra, which is specified as a tree of java objects - similar to many other SQL engines (Hive, Flink, ...)
 - convert this description of the query from java objects into dask API calls (and execute them) - returning a dask dataframe.
 
-For the first step, Apache Calcite needs to know about the columns and types of the dask dataframes, therefore some java classes to store this information for dask dataframes are defined in `planner`.
-After the translation to a relational algebra is done (using `RelationalAlgebraGenerator.getRelationalAlgebra`), the python methods defined in `dask_sql.physical` turn this into a physical dask execution plan by converting each piece of the relational algebra one-by-one.
+For the first step, Apache Datafusion needs to know about the columns and types of the dask dataframes, therefore some Rust code to store this information for dask dataframes are defined in `dask_planner`.
```

Comment:
DataFusion isn't a top-level Apache project but part of the Apache Arrow project, so we should probably refer to it either as just "DataFusiuon" or by the full name "Apache Arrow DataFusion".

```suggestion
For the first step, Apache Arrow DataFusion needs to know about the columns and types of the dask dataframes, therefore some Rust code to store this information for dask dataframes are defined in `dask_planner`.
```